frontend/src/pages/MetadataByType/MetadataByTypeResults.jsx

```diff
@@ -121,7 +121,7 @@ const MetadataByTypeResults = ({ metadataTypeUuid }) => {
       </ResultsHeader>
       <ResultsContainer>
         { metadataSet.length === 0 ? (
-          <NoResult text={`No results were found for ${metadataTypeUuid}.`} />
+          <NoResult text={`No results were found for ${metadataType.name}.`} />
```

Comment:
```suggestion
          <NoResult text={`No results were found for ${name}.`} />
```
The name variable is already defined. Should be this instead.